src/ILCompiler.ReadyToRun/src/IL/ReadyToRunILProvider.cs

```diff
@@ -74,3 +209,16 @@ public override MethodIL GetMethodIL(MethodDesc method)
         }
     }
 }
+
+namespace Internal.IL.Workarounds
+{
+    static class IntrinsicExtensions
+    {
+        // We should ideally mark interesting methods a [Intrinsic] to avoid having to
```

Comment:
Could you please mark the missing methods in CoreCLR CoreLib? Most of them should be marked already.

Reply:
Yup, dotnet/coreclr#26209.